arpwatch.te

```diff
@@ -24,6 +24,8 @@ files_pid_file(arpwatch_var_run_t)
 type arpwatch_unit_file_t;
 systemd_unit_file(arpwatch_unit_file_t)
 
+type mail_home_rw_t;
```

Comment:
SELinux type mail_home_rw_t is declared here: https://github.com/fedora-selinux/selinux-policy-contrib/blob/rawhide/mta.te#L26. This should be removed.